cudaq/buildspec.yml

```diff
@@ -0,0 +1,38 @@
+account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
+region: &REGION <set-$REGION-in-environment>
+framework: &FRAMEWORK cudaq
+version: &VERSION 1.0
```

Comment:
Version is based on the dependency version. See: https://github.com/amazon-braket/amazon-braket-containers/blob/main/tensorflow/buildspec.yml#L4

Reply:
The version should reflect the cudaq version